src/manage_sql.c

```diff
@@ -54400,6 +54421,14 @@ update_from_slave (task_t task, entity_t get_report, entity_t *report,
                                     entity_text (host_start));
         }
       hosts = next_entities (hosts);
+
+      current_commit_size ++;
+      if (slave_commit_size && current_commit_size >= slave_commit_size)
+        {
+          sql_commit();
```

Comment:
Missing space.

Reply:
The missing space has been added in fd2fb81.